client/src/app.js

```diff
@@ -207,6 +207,10 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
   // In elements, we block rendering until the assetMap is loaded. Otherwise, we can start immediately.
   , isReady$ = process.env.ASSET_MAP_URL ? assetMap$.mapTo(true).startWith(false) : O.of(true)
 
+  // Asset Icons Response
+  , assetIcons$ = !process.env.ASSET_MAP_URL ? O.of({}) :
```

Comment:
This is independent of `ASSET_MAP_URL`, should use `IS_ELEMENTS` or the `ASSET_ICONS_URL` configuration proposed below.

Reply:
Changes made